### PR TITLE
fix for returning current souce rather than blob

### PR DESCRIPTION
**Pull request by bcvio**

### CODE REVIEW

**Comment by pam** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: 339f8675cb580bd1d258ec30a1e34ac0a5ffaa31
Build details: https://travis-ci.org/pam/video.js/builds/92315225

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by gkatsev** [comment]
LGTM

**Comment by dmlap** [comment]
Could use a test but otherwise looks good.

**Comment by pam** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: e3c311c32c929b95b12da0612d2ca6284f9971e0
Build details: https://travis-ci.org/pam/video.js/builds/92569784

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

File: test/unit/tech/html5.test.js [hunk 1]
```diff
@@ -249,6 +249,12 @@ if (Html5.supportsNativeTextTracks()) {
     equal(adds[2][0], rems[2][0], 'removetrack event handler removed');
   });
 }
+test('should always return currentSource_ if set', function(){
+  let currentSrc = Html5.prototype.currentSrc;
+  equal(currentSrc.call({el_: {currentSrc:"test1"}}), "test1", "sould return source from element if nothing else set");
```
Comment (gkatsev) [inline test/unit/tech/html5.test.js]: Build is failing. These should be single quotes.

**Comment by pam** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: bf310328f69671c3223fd2b1eb5a6e489ba7e98c
Build details: https://travis-ci.org/pam/video.js/builds/92746315

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by gkatsev** [comment]
@pam retry

**Comment by pam** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: bf310328f69671c3223fd2b1eb5a6e489ba7e98c
Build details: https://travis-ci.org/pam/video.js/builds/92750754

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Pull request closed without merging**